supervisor/updater.py

```diff
@@ -243,6 +243,15 @@ async def fetch_data(self):
                 self._data[ATTR_OTA] = data["ota"]
                 if version := data["hassos"].get(self.sys_os.board):
                     events.append("os")
+                    upgrade_map = data.get("hassos-upgrade", {})
+                    if last_in_major := upgrade_map.get(str(self.sys_os.version.major)):
```

Comment:
`.major` returns `None` for non SemVer versions (like our OS)
use `.section(0)` to get the first part of the version instead

Reply:
Actually, that's dumb.
I'll fix that (support more variants) in the lib

Reply:
https://github.com/home-assistant/supervisor/pull/3634